translations/ui-organizations/en.json

```diff
@@ -284,8 +37,8 @@
   "filterConfig.paymentMethod": "Payment method",
   "filterConfig.statsAvailable": "Stats available",
   "filterConfig.vendorStatus": "Organizations status",
-  "filterConfig.boolean.true": "True",
-  "filterConfig.boolean.false": "False",
+  "filterConfig.boolean.true": "Yes",
+  "filterConfig.boolean.false": "No",
```

Comment:
@dltj please add these changes to the next translation PR. Thanks

Reply:
https://github.com/folio-org/ui-organizations/pull/149